### DISCUSSION TITLE
FDS Hardware Questions - Hyperthreading on or off?, Impacts of RAM speed (2400 MHz vs 3200 MHz)?

### DISCUSSION THREAD

**Discussion by norm07** [firemodels/fds#13740] [Q&A]
Hi All,

Wanted to tweak some settings on my computer to gain a performance bump with FDS as i have a bunch of models to run in the lead up to this years end. 

I've been reading some previous posts regarding turning off hyperthreading - can i confirm that this still yields a significant improvement in processing speed.

Lastly, my RAM sticks can be overclocked to 3200 MHz using intel X.M.P in the BIOS, i was wondering whether this was also worth enabling for a performance boost.

Kind regards,
N.

**Comment by mcgratta** [comment]
We have found that hyperthreading does not help. We turn it off. Not sure about RAM speed.
